packages/nodes-base/nodes/Transform/RemoveDuplicates/test/RemoveDuplicates.test.ts

```diff
@@ -1,5 +1,66 @@
+import type { INode } from 'n8n-workflow';
+import { validateInputData } from '../utils';
 import { testWorkflows, getWorkflowFilenames } from '@test/nodes/Helpers';
 
 const workflows = getWorkflowFilenames(__dirname);
 
 describe('Test Remove Duplicates Node', () => testWorkflows(workflows));
+
+describe('Test Remove Duplicates Node, validateInputData util', () => {
+	test('Should throw error for version 1', () => {
+		try {
+			validateInputData(
+				{
+					name: 'Remove Duplicates',
+					type: 'n8n-nodes-base.removeDuplicates',
+					typeVersion: 1,
+				} as INode,
+				[
+					{ json: { country: 'uk' } },
+					{ json: { country: 'us' } },
+					{ json: { country: 'uk' } },
+					{ json: { country: null } },
+				],
+				['country'],
+				false,
+			);
+		} catch (error) {
+			expect(error.message).toEqual("'country' isn't always the same type");
```

Comment:
I think this test would still succeed if `validateInputData` would not throw (because there would be no `expect`)

You can use `expect(() => validateInputData(...)).toThrow(...)`